Library/Homebrew/cask/lib/hbc/dsl/caveats.rb

```diff
@@ -125,6 +125,13 @@ def eval_caveats(&block)
         EOS
       end
 
+      caveat :license do |web_page|
+        <<~EOS
+          Installing this Cask means you have AGREED to the license at:
```

Comment:
For consistency with the other messages, use `#{@cask}` and remove `:`.

```
Installing #{@cask} means you have AGREED to the license at
```

Reply:
Updated.